src/clr-angular/data/datagrid/render/cell-renderer.ts

```diff
@@ -4,15 +4,33 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import { Directive, ElementRef, OnDestroy, Renderer2 } from '@angular/core';
-import { Subscription } from 'rxjs';
+import { BehaviorSubject, Subscription } from 'rxjs';
 
 import { DatagridRenderStep } from '../enums/render-step.enum';
 
 import { STRICT_WIDTH_CLASS } from './constants';
 import { DatagridRenderOrganizer } from './render-organizer';
+import { DatagridColumnState } from '../interfaces/column-state.interface';
+import { DatagridColumnChanges } from '../enums/column-changes.enum';
 
 @Directive({ selector: 'clr-dg-cell' })
 export class DatagridCellRenderer implements OnDestroy {
+  private stateSubscription: Subscription;
+  private _columnState: BehaviorSubject<DatagridColumnState>;
+
+  get columnState(): BehaviorSubject<DatagridColumnState> {
```

Comment:
Do we need to have a getter for this? Nothing should be subscribing to from outside, right?

Reply:
The reason for this was that you could get the column state from here, though it is not in use right now. I can remove this for now unless it becomes needed.